update_version.sh

```diff
@@ -7,9 +7,7 @@ dch_installed=$?
 command -v git > /dev/null
 git_installed=$?
 if [ $dch_installed -ne 0 ] || [ $git_installed -ne 0 ]; then
-  echo "You must run this on a system with dch and git available (in order to edit the Debian package changelog and commit message)"
-  echo "If you are on Debian/Ubuntu, apt-get install devscripts git"
-  exit 1
+  sudo apt-get install -y devscripts git
```

Comment:
Minor: If you're just gonna install the dependencies, you can probably remove or at least simplify the conditionals that check if those dependencies are already installed. For example, `apt-get install -y` won't do anything wrong if the packages you want are already installed. [Some ideas from StackOverflow](http://stackoverflow.com/questions/1298066/check-if-a-package-is-installed-and-then-install-it-if-its-not).